docs/gaia/validators/validator-setup.md

```diff
@@ -22,32 +60,63 @@ Your `cosmosvalconspub` can be used to create a new validator by staking tokens.
 gaiad tendermint show-validator
 ```
 
-Next, craft your `gaiacli tx stake create-validator` command:
+Next, craft your `gaiad gentx` command:
 
 ::: warning Note
 Don't use more `steak` thank you have! You can always get more by using the [Faucet](https://faucetcosmos.network/)!
 :::
 
 ```bash
-gaiacli tx stake create-validator \
-  --amount=5steak \
-  --pubkey=$(gaiad tendermint show-validator) \
-  --moniker="choose a moniker" \
-  --chain-id=<chain_id> \
-  --from=<key_name> \
-  --commission-rate="0.10" \
-  --commission-max-rate="0.20" \
-  --commission-max-change-rate="0.01"
+gaiad gentx \
+  --amount <amount_of_delegation> \
+  --commission-rate <commission_rate> \
+  --commission-max-rate <commission_max_rate> \
+  --commission-max-change-rate <commission_max_change_rate> \
+  --pubkey <consensus_pubkey> \
+  --name <key_name>
 ```
 
 __Note__: When specifying commission parameters, the `commission-max-change-rate`
 is used to measure % _point_ change over the `commission-rate`. E.g. 1% to 2% is
 a 100% rate increase, but only 1 percentage point.
 
+__Note__: If unspecified, `consensus_pubkey` will default to the output of `gaiad tendermint show-validator`.
+`key_name` is the name of the private key that will be used to sign the transaction.
+
 ::: tip
-Use `gaiacli tx stake create-validator -h` to get a list of all the available flags.
+Consult `gaiad gentx --help` for more information on the flags defaults.
 :::
 
+#### What are genesis transactions for?
+
+A `genesis transaction` is a JSON file carrying a self-delegation. All genesis transactions are collected by a `genesis coordinator` and validated against an initial `genesis.json`. Such initial `genesis.json` contains only a list of accounts and their coins. Once the transactions are processed, they are merged in the `genesis.json`'s `gentxs` field.
+
+#### Copy the Initial Genesis File and Process Genesis Transactions
+
+Fetch the `genesis.json` file into `gaiad`'s config directory.
+
+```bash
+mkdir -p $HOME/.gaiad/config
+curl https://raw.githubusercontent.com/cosmos/testnets/master/latest/genesis.json > $HOME/.gaiad/config/genesis.json
+```
+
+Note we use the `latest` directory in the [testnets repo](https://github.com/cosmos/testnets)
+which contains details for the latest testnet. If you are connecting to a different testnet, ensure you get the right files.
+
+Once you've collected all genesis transactions in `~/.gaiad/config/gentx`, you can run:
+
+```bash
+gaiad collect-gentxs
+```
+
+The previous command will collect all genesis transactions and modify `genesis.json` accordingly.
```

Comment:
Aren't we missing a step here? Once `genesis.json` has been modified, what's next? Every validator/node need to start from the same `genesis.json`, so this modified `genesis.json` needs to be somehow broadcasted to other validators. Is the expected process be that after you run `collect-gentxs` you open a PR against the `genesis.json` file on github to update it? If so, it should be indicated.

Reply:
That's correct. In fact, a gensis coordinator will run `collect-gentxs` to generate and share the new `genesis.json`

Reply:
This should be indicated in the file then

Reply:
I've amended the wording. Please review

Reply:
Wait, wdym by "generate and share"? It's only the generation part if I'm not mistaking. The sharing must happen on github or something right? If yes, then it should be specified "Once `genesis.json` is modified to include your `GenTxs`, open a PR on the testnet repo to..."

Reply:
Unsure about what will be the procedure, @zmanian: how do we go about it?